react/week4/my-app/src/index.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import FindUser from "./FindUser";
+import context from "./context";
+
+const ContextProvider = context.Provider;
+
+class App extends React.Component {
+  state = { userName: "", usersList: [], isLoading: false };
+  handelUserSearch = this.handelUserSearch.bind(this);
+
+  handelUserSearch(e) {
+    const value = e.target.value;
+    this.setState({ userName: value });
```

Comment:
I think you could get rid of the `userName` state. You don't do anything with it other than set its value. 😄